sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2861,14 +2861,14 @@ object GpuOverrides extends Logging {
     expr[MapFromArrays](
       "Creates a new map from two arrays",
       ExprChecks.binaryProject(
-        TypeSig.MAP.nested(TypeSig.commonCudfTypes),
-        TypeSig.MAP.nested(TypeSig.commonCudfTypes),
-        ("array",
-          TypeSig.ARRAY.nested(TypeSig.commonCudfTypes),
-          TypeSig.all),
-        ("array",
-          TypeSig.ARRAY.nested(TypeSig.commonCudfTypes),
-          TypeSig.all)),
+        TypeSig.MAP.nested(TypeSig.all - TypeSig.MAP),
```

Comment:
But do we really support all of the types?  What about CalendarInterval?

Reply:
Updated, thank you!